crates/ruff_linter/src/rules/flake8_simplify/rules/split_static_string.rs

```diff
@@ -136,22 +135,27 @@ fn replace_flags(elt: &str, flags: StringLiteralFlags) -> StringLiteralFlags {
     // 'single'quoted
     // """.split() # -> [r"itemA",r"'single'quoted'"]
     // ```
-    if !flags.prefix().is_raw() || !elt.contains(flags.quote_style().as_char()) {
-        flags.with_triple_quotes(TripleQuotes::No)
+
+    if !flags.prefix().is_raw() {
+        return flags.with_triple_quotes(ruff_python_ast::str::TripleQuotes::No);
+    }
+
+    if elt.contains('\n') || elt.contains('\r') {
```

Comment:
nit: I think we could use a single `contains` call here:


```suggestion
    if elt.contains(['\n', '\r']) {
```